.github/workflows/update_image.yml

```diff
@@ -0,0 +1,23 @@
+name: Update the antrea-operator and bundle images every 12 hours
```

Comment:
Do you think the periodic job will bring additional benefits?
I was thinking that perhaps updating the 'latest' tag everytime a PR is merged could be enough.

Reply:
Thanks Salvatore, I think we could remove it as you said.